### PR TITLE
UI: apply timezone settings

### PR DESCRIPTION
**Pull request by ysknkd**
## Background
At present, the time displayed in the UI is set to UTC by default. Consequently, users of the UI are required to individually convert the displayed time to their respective local time zones, which can be rather inconvenient.

## Overview
In this pull request, I intend to modify the UI so that the displayed time aligns with the server's TimeZone setting, thereby enhancing user experience.

To achieve this, the React components have been updated to obtain TimeZone settings from Redux. I was uncertain about the most appropriate Redux Store for storing the TimeZone settings and ultimately decided on the `domains` store. If you believe there is a more suitable location for these settings, kindly share your insights.

Additionally, as the TimeZone will be utilized across the majority of pages within the UI, I have incorporated the necessary changes into `_app.js`. If there is a more efficient or effective approach to implementing this feature, please do not hesitate to share your suggestions.

### CODE REVIEW

**Comment by ysknkd** [comment]
I apologize for the large commit. If using redux is not appropriate, I'm thinking of exploring other methods. I would appreciate any comments you may have.

**Comment by abvaidya** [comment]
Apologies for the delay. Our team has started looking into it but they haven't completed the review yet. I will check with them this week. Thank you for your patience :)

File: ui/src/components/member/Member.js [hunk 1]
```diff
@@ -71,11 +73,11 @@ export default class Member extends React.Component {
     render() {
         let exp = this.props.item.expiration;
         if (exp) {
-            exp = this.localDate.getLocalDate(exp, 'UTC', 'UTC');
+            exp = this.localDate.getLocalDate(exp, this.props.timeZone, this.props.timeZone);
         }
         let review = this.props.item.reviewReminder;
         if (review) {
-            review = this.localDate.getLocalDate(review, 'UTC', 'UTC');
+            review = this.localDate.getLocalDate(exp, this.props.timeZone, this.props.timeZone);
```
[the remark this answers is not in the document]
Comment (ysknkd) [reply ui/src/components/member/Member.js]: @mendi160 Thank you for your confirmation. I've fixed it.

**Review by mendi160 — COMMENTED** [review]
Screenshot 2023-06-02 at 12 20 14
My time zone is IDT (Isreal), I get a different time zone (maybe VPN cause).
What I don't understand is why I get a different time zone for the second member's expiration and second member's review reminder

**Comment by havetisyan** [comment]
> Screenshot 2023-06-02 at 12 20 14 My time zone is IDT (Israel) and I get a different time zone. What I don't understand is why I don't get the same time zone for the second member's expiration and the second member's review reminder

@mendi160 that's actually correct and is not a different time zone. Both are Pacific Time Zone - however, based on the date, it correctly reports the daylight savings time in effect on that date:
PDT - Pacific Daylight Time zone
PST - Pacific Standard Time zone

**Review by mendi160 — COMMENTED** [review]
@ysknkd I couldn't understand how it works. I saw that in the end, you take the timezone from the config file, but how it is set with the user time zone? please point me to what I'm missing, thanks

**Comment by ysknkd** [comment]
@mendi160 I apologize for the confusion, my explanation was not clear. This PR does not change the display according to each user's timezone. In our environment, we basically operate in JST and our users are also familiar with JST. Therefore, we wanted to convert the timezone displayed in the UI to JST, which led to this change.

**Comment by mendi160** [comment]
> @mendi160 I apologize for the confusion, my explanation was not clear. This PR does not change the display according to each user's timezone. In our environment, we basically operate in JST and our users are also familiar with JST. Therefore, we wanted to convert the timezone displayed in the UI to JST, which led to this change.

If so, it LGTM

**Review by mendi160 — APPROVED** [review]

**Review by mendi160 — APPROVED** [review]

**Pull request merged**